warehouse/query-core/src/main/java/datawave/query/jexl/lookups/BoundedRangeIndexLookup.java

```diff
@@ -50,7 +51,7 @@ public class BoundedRangeIndexLookup extends AsyncIndexLookup {
     private final LiteralRange<?> literalRange;
 
     protected Future<Boolean> timedScanFuture;
-    protected long lookupStartTimeMillis = Long.MAX_VALUE;
+    protected AtomicLong lookupStartTimeMillis = new AtomicLong(Long.MAX_VALUE);
```

Comment:
Was this because of thread access issues?  Could have used the "volatile" qualifier.

Reply:
ah, it was more because of the reference part.... got it.